service/writeVersionProperties.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Input:
+# $1 - version - the version number from inside gradle
+# $2 - output file to write
+#
```

Comment:
Actually, this might be an opportunity to write our first plugin based on this script.